mobile/src/store/middlewares/notify-logger/notify-logger.ts

```diff
@@ -0,0 +1,18 @@
+import type { MiddlewareAPI, Middleware } from '@reduxjs/toolkit';
+import { NotificationType } from '~/common/enums/enums';
+import { notify } from '~/store/auth/actions';
+
+const notifyLogger: Middleware =
+  ({ dispatch }: MiddlewareAPI | any) =>
```

Comment:
We can avoid any here

```suggestion
  ({ dispatch }: HandleErrorParams) =>
```
```
type HandleErrorParams = {
  dispatch: AppDispatch;
};
```